buildconfig/pygame-stubs/surface.pyi

```diff
@@ -95,21 +73,19 @@ class Surface(object):
     @overload
     def subsurface(
         self,
-        left_top: Union[List[float], Tuple[float, float], Vector2],
-        width_height: Union[List[float], Tuple[float, float], Vector2],
+        left_top: Union[Sequence[float], Tuple[float, float], Vector2],
+        width_height: Union[Sequence[float], Tuple[float, float], Vector2],
     ) -> Surface: ...
     @overload
-    def subsurface(
-        self, left: float, top: float, width: float, height: float
-    ) -> Surface: ...
+    def subsurface(self, left: float, top: float, width: float, height: float) -> Surface: ...
     def get_parent(self) -> Surface: ...
     def get_abs_parent(self) -> Surface: ...
     def get_offset(self) -> Tuple[int, int]: ...
     def get_abs_offset(self) -> Tuple[int, int]: ...
     def get_size(self) -> Tuple[int, int]: ...
     def get_width(self) -> int: ...
     def get_height(self) -> int: ...
-    def get_rect(self, **kwargs) -> Rect: ...
+    def get_rect(self, **kwargs: Any) -> Rect: ...  # this needs to be Any. Applied to any attribute. eg. center - Tuple
```

Comment:
I don't understand this :) Could you please elaborate?

Reply:
Removing the Any